hazelcast/src/main/resources/hazelcast-client-full-example.xml

```diff
@@ -306,18 +306,32 @@
             </properties>
         </ssl>
         <auto-detection enabled="false"/>
-        <aws enabled="true" connection-timeout-seconds="11">
-            <inside-aws>true</inside-aws>
-            <access-key>TEST_ACCESS_KEY</access-key>
-            <secret-key>TEST_SECRET_KEY</secret-key>
-            <region>us-east-1</region>
+        <aws enabled="false">
+            <access-key>my-access-key</access-key>
+            <secret-key>my-secret-key</secret-key>
+            <region>us-west-1</region>
             <host-header>ec2.amazonaws.com</host-header>
-            <security-group-name>hazelcast-sg</security-group-name>
+            <connection-timeout-seconds>7</connection-timeout-seconds>
+            <read-timeout-seconds>7</read-timeout-seconds>
+            <connection-retries>4</connection-retries>
+            <hz-port>5701-5710</hz-port>
+            <use-public-ip>true</use-public-ip>
             <tag-key>type</tag-key>
             <tag-value>hz-nodes</tag-value>
+            <security-group-name>hazelcast-sg</security-group-name>
+            <iam-role>dummy</iam-role>
+            <cluster>my-cluster</cluster>
```

Comment:
I'd probably comment-out `cluster`, `family`, and `service-name` since these properties are exclusive with the properties above. 

The same for YAML and member config.

Reply:
Yeah at first I had them seperated with comments showing their mutually exclusivity. But shouldn't full-example yamls and xmls have all the properties defined and the user should refer to the plug-in repository to learn more about on how to use them? I tried to have all the logic explained with comments but it seemed unnecessarily long.

Reply:
No need to explain anything in the comments. I'd just comment-out these 3 properties. There is a lot of other commented-out configuration in `*-full-*` configurations, so I guess it's fine to do the same.